charts/common/templates/deployment.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ include "common.name" . }}
   labels:
     tags.datadoghq.com/version: "{{ .Values.image.tag }}"
+    run: {{ include "common.name" . }}
```

Comment:
maybe an option is to handle this one here?
https://github.com/dave-inc/charts/blob/master/charts/common/templates/_helpers.tpl#L22

Reply:
Leaving it at this leve right now to tag only deployments and pods, we can revisit this in case we need the label propagated to the rest of the elements, will check back with requester once he is back from PTO